### PR TITLE
Assignment1 @A-Tanner

### PR DESCRIPTION
**Pull request by Disciple4GIT**

### CODE REVIEW

**Comment by hewrtarkhany** [comment]
can you include the URL link for your website?

File: wordle-web/pages/game.vue [hunk 1]
```diff
@@ -12,11 +12,11 @@
       </blockquote>
       <v-card>
         <v-card-title class="headline">
-          Werlcome to the start of something medicore. 
+          Welcome to the start of the Next Big Thing™ 
         </v-card-title>
         <v-card-text>
           <div class="text-xs-right">
-            <em><small>&mdash; Hey We finally go it working!!!! NEW Game coming soon, so stay tuned!</small></em>
+            <em><small>The world of real time competitive multiplayer Wordle is about to arrive...</small></em>
```
Comment (hewrtarkhany) [inline wordle-web/pages/game.vue:19]: Nice, but I do not see your URL, so i can see if your webpage is working. thanks

**Review by hewrtarkhany — CHANGES_REQUESTED** [review]

**Review by linhtran8699 — COMMENTED** [review]
Everything looks good to me. Great job!

**Comment by mmwoodfo** [comment]
In the future, just make sure that you are making a separate branch for your assignment, rather than making a PR using your main. Code changes showing up here in the PR should only be relevant to the assignment the PR is for.

Here is an example of a PR that uses an Assignment 1 branch: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/11

Great job!

**Review by mmwoodfo — COMMENTED** [review]
I don't see a link for the Azure app here!

**Comment by mmwoodfo** [comment]
I think this is it? Found it in your actions pipeline output: https://thankful-water-09f62b010.1.azurestaticapps.net/

Make sure to include a link to this in the PR comments in the future.

**Comment by mmwoodfo** [comment]
@Disciple4GIT I don't see a PR review. Let me know if you did one and I missed it.

**Pull request closed without merging**